gps/app/views/Licenses.js

```diff
@@ -1,25 +1,24 @@
+/* eslint-disable react-native/no-raw-text */
 import React, { useEffect } from 'react';
```

Comment:
this file has been copied from `1.0.65`.
I also added a safe area view for the bottom button